### PR TITLE
Rename CypressDev to CypressOnRails

### PR DESCRIPTION
**Pull request by grantspeelman**
as mentioned in #17 and #11

### CODE REVIEW

**Comment by grantspeelman** [comment]
@justin808 can you enable travis-ci for the project please

File: README.md [hunk 1]
```diff
@@ -202,7 +202,7 @@ add the following file to cypress
 
 ```js
 // test/cypress/support/on-rails.js
```
Comment (yagudaev) [inline README.md]: should the file itself be called `cypress-on-rails`?

File: cypress-on-rails.gemspec [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
 Gem::Specification.new do |s|
   s.name        = "cypress-on-rails"
-  s.version     = CypressDev::VERSION
+  s.version     = CypressOnRails::VERSION
   s.author      = ["miceportal team", 'Grant Petersen-Speelman']
   s.email       = ["info@miceportal.de", 'grantspeelman@gmail.com']
   s.homepage    = "http://github.com/grantspeelman/cypress-on-rails"
   s.summary     = "Integrates cypress with rails or rack applications"
   s.description = "Integrates cypress with rails or rack applications"
+  s.post_install_message = 'The CypressDev constant is being deprecated and will be completely removed and replaced with CypressOnRails'
```
Comment (yagudaev) [inline cypress-on-rails.gemspec]: nice!

**Comment by grantspeelman** [comment]
Screen Shot 2019-06-21 at 8 08 59 am
Installed the branch on our project, and post message comes up 👍

**Comment by grantspeelman** [comment]
Tested the branch on our CI with only updating the gem and it works without any additional changes.

**Pull request merged**